Podfile.lock

```diff
@@ -22,4 +22,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: f17d4908a725d1f7a00298c625f580e0fa05271b
 
-COCOAPODS: 1.12.0
+COCOAPODS: 1.11.3
```

Comment:
do you want to commit this file with the cocoa pods going down?

Reply:
oops, nope, thanks